core/src/lib.rs

```diff
@@ -101,8 +101,8 @@ pub mod prettier {
 
 mod operations;
 pub use operations::{
-    AdviceInjector, AssemblyOp, DebugOptions, Decorator, DecoratorIterator, DecoratorList,
-    Operation, SignatureKind,
+    opcode_constants::*, AdviceInjector, AssemblyOp, DebugOptions, Decorator, DecoratorIterator,
```

Comment:
I'm a bit hesitant to export this many constants at the root level. Maybe we should export this as:
```rust
pub mod opcodes {
    pub use super::operations::opcode_constants::*;
}
```